tests/yast2_gui/yast2_control_center.pm

```diff
@@ -8,7 +8,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Extend YaST2 UI test yast2-control-center
+# Summary: Make YaST2 UI test yast2-control-center more reliable and avoid timeout error
```

Comment:
please don't describe the change within your commit here but what the test module does in the current state